src/features/defi/providers/foxy/components/FoxyManager/Withdraw/components/Withdraw.tsx

```diff
@@ -195,7 +195,7 @@ export const Withdraw: React.FC<
           if (!estimatedGasCrypto) return
           dispatch({
             type: FoxyWithdrawActionType.SET_APPROVE,
-            payload: { estimatedGasCrypto },
+            payload: { estimatedGasCryptoBaseUnit: estimatedGasCrypto },
```

Comment:
Same with the `const` on line `194`.

Reply:
613a7d8f64553ad701533511899ee87a44227a6d